Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'foundation_emails'
 gem 'gibberish'
 gem 'hiredis'
 gem 'http_accept_language'
-gem 'identity-doc-auth', github: '18F/identity-doc-auth', tag: 'v0.1.0'
+gem 'identity-doc-auth', github: '18F/identity-doc-auth', tag: 'ee8dafb97db4226287f7651cc596f69f59ecf58c' # TODO: use tag
```

Comment:
this line fails the build on purpose so I don't accidentally commit a thing before it's merged